libraries/AC_SafeRTL/AC_SafeRTL.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+#include <AP_Buffer/AP_Buffer.h>
+#include <DataFlash/DataFlash.h>
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Math/AP_Math.h>
+
+#include <bitset>
+#include <vector>
+
+#define SMALL_FLOAT  0.0000001f
+
+#define POSITION_DELTA 2.0f // how many meters to move before appending a new position to return_path
```

Comment:
because these definitions go into the global scope (I think), it would be good to prefix them with "SAFERTL_".